src/lib/modules/blockchain_connector/index.js

```diff
@@ -249,7 +249,7 @@ class BlockchainConnector {
         function checkVersion(next) {
           // TODO: web3_clientVersion method is currently not implemented in web3.js 1.0
           self.web3._requestManager.send({method: 'web3_clientVersion', params: []}, (err, version) => {
-            if (err) {
+            if (err || !version) {
```

Comment:
No error and No version, is it a possible case?

Reply:
Somehow I got that once yesterday. Don't know if I'd be able to reproduce though. 
So many weird bugs I got, gives me PTSD.